__tests__/app.test.js

```diff
@@ -117,6 +117,45 @@ describe("GET all articles", () => {
   });
 });
 
+describe('GET /api/articles?sort_by=x&order=ASC', () => {
+  test('should return all articles sorted by "title" in ascending order', () => {
+    return request(app)
+    .get("/api/articles?sort_by=title&order=ASC")
+    .expect(200)
+    .then((response) => {
+      const articles = response.body.articles;
+      expect(articles).toBeSortedBy("title", { ascending: true });
```

Comment:
Ascending isn't an option in jestSorted, however because you haven't put decending:true in the options it will actually default to ascending